.travis.yml

```diff
@@ -32,13 +50,14 @@ cache:
     - $HOME/.ccache
 
 before_install:
-  - ccache -s
   - mkdir -p "${HOME}/.cache/download"
-  - ci/install-edm.sh
-  - export PATH="${HOME}/edm/bin:/usr/lib/ccache:${PATH}"
-  - edm install -y wheel click coverage
   - export DISPLAY=:99.0
-  - sh -e /etc/init.d/xvfb start
+  - if [[ ${TRAVIS_OS_NAME} == "linux" ]] ; then ccache -s; fi
```

Comment:
(suggestion) It would be more readable to consolidate the if statements as:
```
- |
  if [[ ${TRAVIS_OS_NAME} == "linux" ]]; then
     ccache -s
     sh -e /etc/init.d/xvfb start
     ./ci/install-edm-linux.sh
  else
    ./ci/install-edm-osx.sh
    export PATH="${PATH}:/usr/local/bin"
  fi
```

Reply:
done. the osx + python 2.7 + wx job is still failing though. not exactly what to do about that.

Reply:
Can you try using 'null.image' as the toolkit?

Reply:
sorry, my previous comment was completly wrong

Reply:
I think that for the moment we can allow the `osx-2.7-wx` build to fail and open a separate issue to fix the tests.

Reply:
added osx-2.7-wx as an allowed failure.